### ISSUE TITLE
Add an Animation::ease usecase in Magnum::Animation::Track documentation

### ISSUE THREAD

**Issue by Amphaal** [mosra/magnum#477]
That's something I really craved for and could not find about until I stumbled on it in one of your Gitter post ! (https://tmp.mosra.cz/water/#water-is-life)
IMO that's should be highlighted in the documentation.

**Comment by mosra** [comment]
Yeah the Animation lib is [seriously underdocumented](https://doc.magnum.graphics/magnum/animation.html), I have to fix that.
